rise-playlist.html

```diff
@@ -254,7 +254,7 @@
        */
       hide: function() {
         for (var i = 0; i < this.items.length; i++) {
-          this.items[i].element.style.visibility = "hidden";
+          this.items[i].element.style.display = "none";
```

Comment:
@stulees It still takes up room on the page when you set the visibility of an element. I forgot to change that earlier as I figured it would be problematic.